.github/ISSUE_TEMPLATE/new-feature.md

```diff
@@ -0,0 +1,37 @@
+---
+name: New Feature
+about: Issues for requesting new features on the bot.
+title: ''
+labels: 'status: planning, type: feature'
+assignees: ''
+
+---
+
+## Description
+<!-- Describe the feature you'd like to see added -->
+
+
+## Reasoning
+<!-- Why would the feature be a useful addition to the community -->
+
+
+## Proposed Implementation
+<!-- If you had a way to implement the feature in mind, describe it here, including any command signatures and concept screenshots if possible -->
+
+
+## Possible Alternatives
+<!-- List any possible alternatives to your proposed implementation -->
+
+
+## Possible Issues
```

Comment:
I know this is optional but it seems like it's getting a bit overwhelming with this section. Would it be better to just let this be part of "Additional Details"?

Reply:
Yeah, I added this before I had an Additional Details section. Makes sense that it's unnecessary now.